spec/index.md

```diff
@@ -613,6 +621,27 @@ Calls must stay in `replied` or `rejected` long enough for polling users to catc
 
 When asking the IC about the state or call of a request, the user uses the request id (see [Request ids](#request-id)) to read the request status (see [Request status](#state-tree-request-status)) from the state tree (see [Request: Read state](#http-read-state)).
 
+#### Synchronous canister calling {#http-sync-call}
```

Comment:
I don't get the point of this section - it looks like you copied the first eight steps from the previous section.

Reply:
Instead of repeating the steps, I'd just add a few sentences describing a synchronous call as the replica making an asynchronous call followed by polling for its status on behalf of the caller (until a timeout).

Reply:
I removed the step-by-step points. Can you take another look?